content/docs/dvclive/how-it-works.md

```diff
@@ -81,13 +81,17 @@ find but that don't clutter your Git history or create extra branches.
 ### Track large artifacts with DVC
 
 Models and data are often large and aren't easily tracked in Git.
-`Live.log_artifact("model.pt")` will
+`Live.log_artifact("model.pt", type="model")` will
 [cache](/doc/start/data-management/data-versioning) the `model.pt` file with DVC
 and make Git ignore it. It will generate a `model.pt.dvc` metadata file, which
 can be tracked in Git and becomes part of the experiment. With this metadata
 file, you can [retrieve](/doc/start/data-management/data-versioning#retrieving)
 the versioned artifact from the Git commit.
 
+Passing `type="model"` or `type="data"` will it to `artifacts` section of
```

Comment:
I think we are adding to `artifacts.yaml` regardless of what other info gets passed, right @daavoo? As mentioned in the other comments, let's clarify what gets written to `artifacts.yaml` and clarify that the model registry won't do anything yet but soon will work with `type=model` artifacts.

Reply:
You are right @dberenbaum